src/main/java/io/github/kawamuray/wasmtime/NativeLibraryLoader.java

```diff
@@ -79,10 +79,10 @@ private static String libVersion() throws IOException {
 
     @AllArgsConstructor
     private enum Platform {
-        LINUX("linux","lib" , ".so"),
-        MACOS("macos","lib", ".dylib"),
-        WINDOWS("windows","",".dll")
-        ;
+        LINUX("linux", "lib", ".so"),
+        MACOS("macos", "lib", ".dylib"),
+        MACOS_AARCH64("macos_aarch64", "lib", ".dylib"),
```

Comment:
Maybe we better have `enum Os` and `enum Arch`, and make this `Platform` to hold these, for better organization?